packages/adapter-drizzle/src/lib/mysql.ts

```diff
@@ -130,14 +118,12 @@ export function MySqlDrizzleAdapter(
       userId: string
       expires: Date
     }) {
-      const id = randomUUID()
-
-      await client.insert(sessionsTable).values({ ...data, id })
```

Comment:
this is still a breaking change (if we compare the last 0.x release) that now requires the user to have the `.$defaultFn()` or `.default()`. Is there some way to check if the table has a default? There should be some props on `sessionsTable._` I think

Reply:
(commented on the rong table, meant to comment on the `createUser` call)

Reply:
Now it is not obligatory to have `.$defaultFn()` or `.default()` if we speak about types. We can generate `id` in the code and pass it to insert values or just use `.$defaultFn()` or `.default()` regarding to `id` has default or not.

 `.$defaultFn(() => randomUUID())` is the same as

```ts
const id = randomUUID()

await client.insert(sessionsTable).values({ ...data, id })
```

Reply:
it's not quite, consider someone doesn't have a default set on their column definition, then inserts would now start failing cause the lib isn't generating one like it used to do.

but if the lib is generating and sending to the insert call, then it doesn't matter if i have or haven't got a default on the table definition since it'll never be used since the param is always passed to the insert values.

so you want to generate an id in the lib code __if and only if__ the column definition doesn't have a `.default()` or `.$defaultFn()` set

Reply:
something like

```ts
if (usersTable._.columns.id.hasDefault) {
  await db.insert(usersTable).values(data)
} else {
  const id = crypto.randomUUID();
  await users.insert({ ...data, id });
}
```

Reply:
So this is still an active breaking change, if I understand correctly, right?

Reply:
I have implemented this workaround:

```ts
      const hasDefaultId = getTableColumns(usersTable)["id"]["hasDefault"]
      
      await client
        .insert(usersTable)
        .values(hasDefaultId ? data : { ...data, id: crypto.randomUUID() })
```